TestProject/GameManager.cs

```diff
@@ -108,19 +108,30 @@ private void EndGameWin(GridState state)
             string winnerName = state.IsPlayer1Playing ? "Player 1 (Crosses)" : "Player 2 (Noughts)";
 
             Console.WriteLine($"Congratulations {winnerName}, you have won the game!!!");
-            Console.WriteLine("\nPress any key to end the game");
 
-            Console.ReadKey();
+            AskUserToContinueOrStopPlaying();
+        }
+
+        private void AskUserToContinueOrStopPlaying()
+        {
+            Console.WriteLine("\nDo you want to stop playing (Y). Press any key to play a new game");
+
+            var response = Console.ReadLine()?.ToUpper();
```

Comment:
Rather than reading the line, instead read the key and check if the response is ConsoleKey.Y
```suggestion
            var response = Console.ReadKey().Key;
```